rag_experiment_accelerator/utils/timetook.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
```

Comment:
datetime.now is not recommended for performance testing or timing code.

Suggest `perf_counter` instead (see https://realpython.com/python-timer/ for details)

Reply:
Thanks Martin, I was not aware of this. replaced.